### PR TITLE
Add task for building on windows

### PR DESCRIPTION
**Pull request by Kapu1178**
Adds a tasks.json containing the build task for 32bit rustg on windows. `CONTROL + SHIFT + B` to build.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+        {
+            "label": "rust: cargo build (win32)",
+            "args": [
+                "build",
+                "--release",
+                "--target",
+                "i686-pc-windows-msvc"
```
Comment (AffectedArc07) [inline .vscode/tasks.json:10]: Should these be merged since its the same flag?

**Review by AffectedArc07 — COMMENTED** [review]
Is it worth adding a build flag for `--features all` given this is to be executed from VSC?

Also a command to build a release version is kinda silly when this is from an IDE, debug build would make more sense since if youre building this from VSC theres likely a reason.

File: .vscode/tasks.json (hunk 1 above)
Comment (ZeWaka) [reply .vscode/tasks.json:10]: 100%

Comment (Kapu1178) [reply .vscode/tasks.json:10]: The task will not run if they are merged.
```
error: unexpected argument '--target i686-pc-windows-msvc' found

  tip: a similar argument exists: '--target'

Usage: cargo.exe build --release --target <TRIPLE>
```

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                "i686-pc-windows-msvc",
+                "--features",
+                "all",
+            ],
+            "command": "cargo",
+            "problemMatcher": [
+                "$rustc"
+            ],
+            "group": "build"
+        },
```
Comment (ZeWaka) [inline .vscode/tasks.json:18]:
```suggestion
	"tasks": [
        {
            "label": "rust: cargo build (win32)",
            "args": [
                "build",
                "--target",
                "i686-pc-windows-msvc",
            ],
            "command": "cargo",
            "problemMatcher": [
                "$rustc"
            ],
            "group": "build"
        },
        {
            "label": "rust: cargo build (win32, all features)",
            "args": [
                "build",
                "--target",
                "i686-pc-windows-msvc",
                "--features",
                "all",
            ],
            "command": "cargo",
            "problemMatcher": [
                "$rustc"
            ],
            "group": "build"
        },
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**